fms-blueprint-compose-zenoh.yaml

```diff
@@ -33,19 +33,17 @@ services:
     volumes:
       - ./config/zenoh/config-router.json5:/zenoh-config.json5
 
-  fms-forwarder:
+  fms-consumer:
     command: "zenoh -c /zenoh-config.json5"
     depends_on:
       fms-zenoh-router:
         condition: service_started
     volumes:
       - ./config/zenoh/config-client.json5:/zenoh-config.json5
 
-  fms-consumer:
+  fms-forwarder:
     command: "zenoh -c /zenoh-config.json5"
     depends_on:
-      influxdb:
```

Comment:
Why did you remove this condition? Because if the database is not present there is no need to have the forwarder running.

Reply:
Nevermind, in a real-world deployment the forwarder would not know directly about the state of the database in the backend either.

Reply:
That was my thinking as well ...